.github/workflows/main.yaml

```diff
@@ -218,8 +218,8 @@ jobs:
         run: |
           ls target/release
           alias wasmer=target/release/wasmer
-          ./target/release/wasmer compile --target=x86_64-apple-darwin tests/integration/cli/assets/qjs.wasm -o qjs_mac_from_linux.wjit
-          #./target/release/wasmer compile --target=x86_64-pc-windows-msvc tests/integration/cli/assets/qjs.wasm -o qjs_win_from_linux.wjit
+          ./target/release/wasmer compile --target=x86_64-apple-darwin lib/c-api/tests/assets/qjs.wasm -o qjs_mac_from_linux.wjit
+          #./target/release/wasmer compile --target=x86_64-pc-windows-msvc lib/c-api/tests/assets/qjs.wasm -o qjs_win_from_linux.wjit
```

Comment:
Note: this change was done because the `tests/integration/cli/assets/qjs.wasm` was a symlink.
And once tested, as it was a existing file was quite hard to figure out what was going on.

Much easier to use directly the real files!